### DISCUSSION TITLE
MRI image on a single plane

### DISCUSSION THREAD

**Discussion by FrancescoLomi** [simnibs/simnibs#430] [7.Creating head models]
Hi everybody,
Do you know if it is possible to use single-planar MRI (i.e., not 3D MRI images) in SimNIBS? 
Thanks!

**Comment by TorgeW** [comment]
This is not possible, everything in SimNIBS is 3D. Charm needs more then one slice to inform the segmentation, the meshing doesn't work on a single slice. Even if you would have a mesh of a single slice you wouldn't been able to simulate the electric field as it is influenced by the slices that are not present in your image.
